### PR TITLE
enhancement to open-cluster-management.io

### PR DESCRIPTION
**Pull request by mikeshng**
- front page enhancement: modified front page to mention other addons
- divide pieces into core components and addons/integrations

@qiujian16 could you please provide some feedback. If you think this is ok, I want to get the doc developers to review it before merging. Thanks.

ocm-full

Signed-off-by: Mike Ng <ming@redhat.com>

### CODE REVIEW

**Comment by swopebe** [comment]
@qiujian16 @mikeshng it would help if you could make sure the doc team (Myself, Chris, Mikela) all could edit your PRs into your branches or forks.... seems on this one, I cannot assign myself or edit so I will make comments.

File: content/_index.md [hunk 1]
```diff
@@ -10,9 +10,24 @@ The Open Cluster Management project consists of several multicluster components,
...
-## Quick start
+## Quick start with core components
 
-Install and create a [Cluster Manager](https://operatorhub.io/operator/cluster-manager) on your _hub_ cluster. For more details see [Install Cluster Manager](getting-started/install-cluster-manager).
+![open-cluster-management](/ocm-small.png)
 
-Install and create a [Klusterlet agent](https://operatorhub.io/operator/klusterlet) on your _managed_ cluster.
-For more details see [Install Klusterlet](getting-started/register-cluster).
\ No newline at end of file
+Install and create a [Cluster Manager](https://operatorhub.io/operator/cluster-manager) on your _hub_ cluster.
```
Comment (swopebe) [inline content/_index.md:17]: sentence case: Cluster manager

File: content/_index.md [hunk 2]
```diff
@@ -10,9 +10,24 @@ The Open Cluster Management project consists of several multicluster components,
...
+
+Install and create a [Klusterlet agent](https://operatorhub.io/operator/klusterlet) on your _manage_ cluster.
+
+For more details see [Core Components](getting-started/core).
+
+## Governance policy framework
+
+Enable the Policy Framework addon which provides governance capability to gain visibility, and drive remediation for various security and configuration aspects to help meet such enterprise standards. For more details see [Policy Framework](getting-started/integration/policy-framework).
```
Comment (swopebe) [inline content/_index.md:25]: comma after addon:

addon, which

Comment (swopebe) [reply content/_index.md:25]: also, this should be add-on

Comment (swopebe) [reply content/_index.md:25]: Please change all addons to add-ons.

Comment (swopebe) [reply content/_index.md:25]: See some of our Word of the day posts:
https://coreos.slack.com/archives/CUHCCV19P/p1607618517043100

**Comment by mikeshng** [comment]
/assign swopebe

@swopebe I am not sure if this will help but we will have to wait for Qiu Jian to be back from vacation to modify access control. I don't have enough permission to add you.

File: content/_index.md [hunk 3]
```diff
@@ -10,9 +10,24 @@ The Open Cluster Management project consists of several multicluster components,
...
+
+## Application lifecycle management
+
+Enable the Application Lifecycle Management addon for delivery, upgrade, and configuration of applications on Kubernetes clusters. For more details see [Application Management](getting-started/integration/app-lifecycle).
```
Comment (swopebe) [inline content/_index.md:29]: Change to: Application lifecycle management add-on

File: content/_index.md [hunk 4]
```diff
@@ -10,9 +10,24 @@ The Open Cluster Management project consists of several multicluster components,
...
+
+## Cluster lifecycle management
+
+Using OKD and Hive technologies, enable the Cluster Lifecycle Management addon to unlock enhanced multiclusters management capabilities. For more details see [Cluster Lifecycle Management](getting-started/integration/cluster-lifecycle).
```
Comment (swopebe) [inline content/_index.md:33]: Cluster lifecycle management add-on

Comment (swopebe) [reply content/_index.md:33]: [Cluster lifecycle management]

Comment (swopebe) [reply content/_index.md:33]: Change: multiclusters
to multicluster

No need for plural or possessive

File: content/getting-started/core/_index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: Core Components
+weight: 1
+---
+
+Install the core control plane of open cluster management that includes cluster registration and manifests distribution on the _hub_ cluster, then install the klusterlet agent on the _manage_ cluster so that it can be registered and managed by the hub cluster.
```
Comment (swopebe) [inline content/getting-started/core/_index.md:6]: I would make this two sentences. They are two steps, really.

File: content/getting-started/core/register-cluster.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [ku
 
 Ensure [golang](https://golang.org/doc/install) is installed, if you are planning to install from the source.
 
-Ensure the open-cluster-management cluster manager is installed on the hub cluster. See [Install Cluster Manager](../install-cluster-manager) for more information.
+Ensure the open-cluster-management cluster manager is installed on the hub cluster. See [Cluster Manager](../cluster-manager) for more information.
```
Comment (swopebe) [inline content/getting-started/core/register-cluster.md:20]: We write open-cluster-management sometimes, then open cluster management... We need to pick one. I mostly see the first option and it works if you want to choose that one.

File: content/getting-started/core/register-cluster.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Install Klusterlet
+title: Klusterlet Agent
```
Comment (swopebe) [inline content/getting-started/core/register-cluster.md:2]: lower case a

File: content/getting-started/core/cluster-manager.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Install Cluster Manager
+title: Cluster Manager
 weight: 1
```
Comment (swopebe) [inline content/getting-started/core/cluster-manager.md:3]: lower case M

File: content/getting-started/integration/policy-framework.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Install Policy Framework
-weight: 5
+title: Policy Framework
```
Comment (swopebe) [inline content/getting-started/integration/policy-framework.md:2]: Lower case f

File: content/getting-started/integration/policy-controllers.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Install Policy Controllers
-weight: 6
+title: Policy Controllers
+weight: 2
```
Comment (swopebe) [inline content/getting-started/integration/policy-controllers.md:3]: lower case c

**Review by swopebe — CHANGES_REQUESTED** [review]
Please make the headers sentence-style, which means only proper names are capitalized and the first word.

File: content/getting-started/integration/app-lifecycle.md [hunk 11]
```diff
@@ -13,12 +13,15 @@ After the cluster manager is installed, you could install the application manage
 
 ## Prerequisite
 
-Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [kustomize](https://kubernetes-sigs.github.io/kustomize/installation/) are installed.
+You must meet the following prerequisites to install the application lifecycle management addon:
```
Comment (swopebe) [inline content/getting-started/integration/app-lifecycle.md:16]: fix name and add-on here, as well

**Comment by mikeshng** [comment]
Thank you Brandi for the review. Added commit https://github.com/open-cluster-management/website/pull/72/commits/6ed94b0e4b0fcd39538e1771ba28335c6a0916cb to address all the issues.

File: content/getting-started/integration/app-lifecycle.md [hunk 12]
```diff
@@ -13,12 +13,15 @@ After the cluster manager is installed, you could install the application manage
...
 
-Ensure the open-cluster-management cluster manager is installed. See [Install Cluster Manager](../install-cluster-manager) for more information.
+* Ensure [golang](https://golang.org/doc/install) is installed, if you are planning to install from the source.
 
+* Ensure the open-cluster-management cluster manager is installed. See [Cluster Manager](/getting-started/core/cluster-manager) for more information.
+
+* Ensure the `open-cluster-management` _klusterlet_ is installed. See [Klusterlet](/getting-started/core/register-cluster) for more information.
 
 ## Install from source
 Clone the `multicloud-operators-subscription`
```
Comment (cadawson) [inline content/getting-started/integration/app-lifecycle.md:27]: Clone the `multicloud-operators-subscription` repository.

File: content/_index.md [hunk 13]
```diff
@@ -10,9 +10,24 @@ The Open Cluster Management project consists of several multicluster components,
 - Define API for policy definition to ensure desired configuration and security settings are auditable or enforceable.
```
Comment (cadawson) [inline content/_index.md:10]: I think it would be easier to follow if ordered lists were used for the procedures.

**Review by cadawson — CHANGES_REQUESTED** [review]
Added a few more comments. Looks good!

**Comment by mikeshng** [comment]
added https://github.com/open-cluster-management/website/pull/72/commits/eb19c57ed9d15999358deb500410ced4638c2bf8 based on Christopher's feedback.

File: content/_index.md [hunk 14]
```diff
@@ -10,9 +10,24 @@ The Open Cluster Management project consists of several multicluster components,
...
 
-Install and create a [Cluster Manager](https://operatorhub.io/operator/cluster-manager) on your _hub_ cluster. For more details see [Install Cluster Manager](getting-started/install-cluster-manager).
+![open-cluster-management](/ocm-small.png)
 
-Install and create a [Klusterlet agent](https://operatorhub.io/operator/klusterlet) on your _managed_ cluster.
-For more details see [Install Klusterlet](getting-started/register-cluster).
\ No newline at end of file
+Install and create a [Cluster Manager](https://operatorhub.io/operator/cluster-manager) on your _hub_ cluster.
+
+Install and create a [Klusterlet agent](https://operatorhub.io/operator/klusterlet) on your _manage_ cluster.
```
[the remark this answers is not in the document]
Comment (mikeshng) [reply content/_index.md:19]: done as suggested. Thanks.

File: content/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mikeshng) [reply content/_index.md]: done as suggested. Thanks.

File: content/getting-started/core/_index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (mikeshng) [reply content/getting-started/core/_index.md]: done as suggested. Thanks.

File: content/getting-started/core/cluster-manager.md [hunk 15]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Install Cluster Manager
+title: Cluster Manager
```
[the remark this answers is not in the document]
Comment (mikeshng) [reply content/getting-started/core/cluster-manager.md]: done as suggested. Thanks.

File: content/getting-started/core/register-cluster.md [hunk 16]
```diff
@@ -17,7 +17,7 @@ Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [ku
 
 Ensure [golang](https://golang.org/doc/install) is installed, if you are planning to install from the source.
 
-Ensure the open-cluster-management cluster manager is installed on the hub cluster. See [Install Cluster Manager](../install-cluster-manager) for more information.
+Ensure the open-cluster-management cluster manager is installed on the hub cluster. See [Cluster Manager](../cluster-manager) for more information.
 
 Prepare another Kubernetes cluster to function as the managed cluster. For example, use [kind](https://kind.sigs.k8s.io/docs/user/quick-start) to create another cluster as below. To use kind, you will need [docker](https://docs.docker.com/get-started) installed and running.
```
[the remark this answers is not in the document]
Comment (mikeshng) [reply content/getting-started/core/register-cluster.md]: done as suggested. Thanks.

File: content/getting-started/integration/app-lifecycle.md (hunk 12 above)
Comment (mikeshng) [reply content/getting-started/integration/app-lifecycle.md]: done as suggested. Thanks.

**Comment by qiujian16** [comment]
/hold
/lgtm

it looks good to me, feel free to unhold

**Comment by mikeshng** [comment]
/unhold

Should be good to merge as all the feedbacks has been addressed.

**Pull request merged**